Cargo.toml

```
@@ -55,6 +55,9 @@ env_logger = "0.11"
 flate2 = "1"
 flume = "0.11"
 futures = "0.3"
+geojson = "0.24.2"
+geojson-vt-rs = "0.2.0"
+geozero = { git = "https://github.com/georust/geozero.git", rev = "9975851" } # TODO: put version in when geozero cuts new release
```

@styrowolf FYI, we finally got this dependency released.
Sorry for the long time

```suggestion
geozero = "0.15.1"
```